blocks/data-list/data-list.js

```diff
@@ -0,0 +1,112 @@
+import { fetchPlaceholders } from '../../scripts/aem.js';
+
+// Fetch placeholders
+const placeholders = await fetchPlaceholders('');
+
+const {
+  sNo, countries, continent, capital, abbreviation,
+} = placeholders;
+
+async function createHeaderDiv(container) {
+  const headerDiv = document.createElement('div');
+  headerDiv.classList.add('header-div');
+
+  const headers = [sNo, countries, continent, capital, abbreviation];
+  headers.forEach((headerText) => {
+    const headerItem = document.createElement('div');
+    headerItem.classList.add('header-item');
+    headerItem.textContent = headerText;
+    headerDiv.append(headerItem);
+  });
+
+  container.append(headerDiv);
+}
+
+async function createRowDiv(container, row, i) {
+  const rowDiv = document.createElement('div');
+  rowDiv.classList.add('row-div');
+
+  const data = [i, row.Country, row.Continent, row.Capital, row.Abbreviation];
+  data.forEach((dataItemText) => {
+    const rowItem = document.createElement('div');
+    rowItem.classList.add('row-item');
+    rowItem.textContent = dataItemText;
+    rowDiv.append(rowItem);
+  });
+
+  container.append(rowDiv);
+}
+
+async function createDivStructure(jsonURL, offset = 0, limit = 20) {
+  const url = new URL(jsonURL);
+  url.searchParams.set('offset', offset);
+  url.searchParams.set('limit', limit);
+
+  console.log(`Fetching data from URL: ${url}`);
+
+  const resp = await fetch(url);
+  const json = await resp.json(); // Fetch and assign the JSON data
+  console.log('Fetched JSON Data: ', json);
```

Comment:
Remove the console and fix all the warnings.

Reply:
removed all console.log